src/OrbitCaptureClient/ApiEventProcessorTest.cpp

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
```

Comment:
As partly mentioned by Florian, I would at least consider using gTest functionalities a bit more.
For example, `ApiEventCaptureListener` seems a good candidate for being a gMock, as you are basically counting how many times `onTimer` has been called with which arguments.
Or `ApiTester` seems to be a good candidate for using a test fixture.
I don't know if it's necessarily better (gMock has a non-negligible learning curve, although it's useful to learn it), but it would at least a more idiomatic use of gTest.